tests/models/mask2former/test_modeling_mask2former.py

```diff
@@ -317,20 +327,30 @@ def prepare_img():
     return image
 
 
+# We will verify our results on a video of cars
+def prepare_video():
```

Comment:
Just checking in with @amyeroberts to see if we should avoid using torchvision over here. For example, VideoMAE [tests](https://github.com/huggingface/transformers/blob/main/tests/models/videomae/test_modeling_videomae.py#LL350C4-L350C4) load a video file converted to a numpy array.